story/stories/examples/DefaultSkinSource.tsx

```diff
@@ -14,7 +15,7 @@ import '@castlabs/prestoplay-react-components/dist/themes/pp-ui-base-theme-embed
 
 const baseConfig = {
   // Replace this by your license key
-  license: '7c8165eb7231508ce4eae02181d53b1c',
+  license: '7c816...81d53b1c',
```

Comment:
Maybe we add a comment here that leaving this out will use the embedded dev license or even make it empty and just have the comment so it "just works" for local development?